.github/workflows/sycl-nightly.yml

```diff
@@ -80,6 +87,37 @@ jobs:
             image_options: -u 1001 --device=/dev/dri --privileged --cap-add SYS_ADMIN
             target_devices: opencl:cpu
             tests_selector: cts
+
+          - name: E2E tests with dev igc on Intel Arc A-Series Graphics
```

Comment:
Please update your commit message with the summary of changes related to Arc GPU testing. @sarnex and @YuriPlyakhin would have to approve the changes to the devigc usage.

Reply:
It looks like in the current PR we start testing on arc with dev igc. IMO if we are going to test arc we shouldn't use dev igc. Today it seems we don't run GPU tests on arc at all, only CPU. I don't think we need to add GPU arc testing as part of this PR.

Reply:
It seems as part of this PR we stop doing dev igc testing in precommit and only do it in the nightly. This is fine with me but we need @jsji's approval, he worked on the dev igc CI and is more familiar with the use case.

Reply:
@ldrumm , why do you suggest to remove dev igc testing from precommit?

Reply:
I think the time spent in running the dev igc testing is fairly small, we should try to keep it in pre-commit if it is possible -- moving it from pre-commit to post-commit (nightly) would make things complicated unless we can  identify the culprit commit and ping author automatically in post-commit (nightly).

Reply:
+1 to what @jsji said

Reply:
The time testing the dev driver might be small but it results in an instability. The principle here is to have a stable testing platform where the user's PR is tested against a stable config, not where a users PR is tested against unstable GPU drivers. If we're to have any unstable test plastforms available in pre-commit they should be explicitly opt-in only

Reply:
It basically boils down to "GPU driver developers should do their own testing, not experiment on dpc++ contributors"

Reply:
Also, @ldrumm , dev igc driver is being updated only after Joint Matrix/ESIMD E2E tests passed with it, so it is stable in context of precommit testing.